python/cog/logging.py

```diff
@@ -86,4 +86,5 @@ def setup_logging(*, log_level: int = logging.NOTSET) -> None:
 
     # Reconfigure log levels for some overly chatty libraries
     logging.getLogger("uvicorn.access").setLevel(logging.WARNING)
+    # FIXME: no more urllib3(?)
```

Comment:
Does `httpx` log to a system logger for normal requests? If not, you can just remove this.

Reply:
looks like it https://www.python-httpx.org/logging/